samples/slack-spring-boot-starter-sample/src/main/kotlin/com/kreait/slack/sample/rock_paper_scissors/RockPaperScissorsChannelListener.kt

```diff
@@ -0,0 +1,59 @@
+package com.kreait.slack.sample.rock_paper_scissors
+
+import com.kreait.slack.api.SlackClient
+import com.kreait.slack.api.contract.jackson.SlackEvent
+import com.kreait.slack.api.contract.jackson.common.messaging.Block
+import com.kreait.slack.api.contract.jackson.common.messaging.Element
+import com.kreait.slack.api.contract.jackson.common.messaging.composition.Text
+import com.kreait.slack.api.contract.jackson.group.chat.PostEphemeralRequest
+import com.kreait.slack.broker.receiver.EventReceiver
+import com.kreait.slack.broker.store.Team
+import com.kreait.slack.sample.rock_paper_scissors.data.WEAPONS
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpHeaders
+import org.springframework.stereotype.Service
+
+@Service
+class RockPaperScissorsChannelListener @Autowired constructor(private val rpsGameHandler: RPSGameHandler,
+                                                              private val slackClient: SlackClient) : EventReceiver {
+
+    override fun supportsEvent(slackEvent: SlackEvent): Boolean {
+        return slackEvent.event["type"] == "message"
```

Comment:
should be more detailed, every message would be handled by this receiver. 
please split this receiver up into one that handles single "rock" "paper" or "scissors" messages, and one that handles the whole sentence.